drvregistry/drvregistry.go

```diff
@@ -164,10 +164,10 @@ func (r *DrvRegistry) RegisterDriver(ntype string, driver driverapi.Driver, capa
 	}
 
 	r.Lock()
-	_, ok := r.drivers[ntype]
+	dd, ok := r.drivers[ntype]
 	r.Unlock()
 
-	if ok {
+	if ok && dd.driver.IsBuiltIn() {
```

Comment:
For pluginv1, this will cause confusion since there is no removal of the older instance. So at a time, registering 2 instances of the same plugin will not cause an error anymore.